node/dataSources/index.ts

```diff
@@ -1,26 +1,34 @@
 import { LRUCache } from '@vtex/api'
+
 import { CatalogDataSource } from './catalog'
 import { CheckoutDataSource } from './checkout'
 import { DocumentDataSource } from './document'
+import { IdentityDataSource } from './identity'
+import { LicenseManagerDataSource } from './licenseManager'
+import { PaymentsDataSource } from './payments'
 import { PortalDataSource } from './portal'
+import { ProfileDataSource } from './profile'
 import { SessionDataSource } from './session'
 import { SubscriptionsDataSource } from './subscriptions'
 import { SubscriptionsGroupDataSource } from './subscriptionsGroup'
-import { ProfileDataSource } from './profile'
-import { PaymentsDataSource } from './payments'
+import { TelemarketingDataSource } from './telemarketing'
```

Comment:
O que de usarmos o termo "Callcenter"?

Ele é usado no Help e em outras aplicações:

https://help.vtex.com/en/faq/order-in-the-name-of-a-customer-using-the-callcenter